CA_DataUploaderLib/CommandHandler.cs

```diff
@@ -355,20 +357,20 @@ private void OnUserCommandAccepted(string cmdString)
                     if (confirmedStop)
                         return "escape";
                 }
-                else if (info.Key == ConsoleKey.Backspace)
+                else if (info.Value.Key == ConsoleKey.Backspace)
                 {
                     Console.Write("\b \b"); // https://stackoverflow.com/a/53976873/66372
                     if (inputCommand.Length > 0)
                         inputCommand.Remove(inputCommand.Length - 1, 1);
                 }
-                else if (info.Key == ConsoleKey.UpArrow)
+                else if (info.Value.Key == ConsoleKey.UpArrow)
                     RestorePreviousAcceptedCommand();
-                else if (info.Key == ConsoleKey.DownArrow)
+                else if (info.Value.Key == ConsoleKey.DownArrow)
                     RestoreNextAcceptedCommand();
                 else
                 {
-                    Console.Write(info.KeyChar);
-                    inputCommand.Append(info.KeyChar);
+                    Console.Write(info.Value.KeyChar);
+                    inputCommand.Append(info.Value.KeyChar);
                 }
                 info = Console.ReadKey(true);
```

Comment:
remember to update other users of ReadKey

Reply:
done